js/qz-tray.js

```diff
@@ -1159,7 +1159,7 @@ var qz = (function() {
              * Any responses from the device will be sent to serial callback functions.
              *
              * @param {string} port An open port to send data over.
-             * @param {Object} data The object containing data and metadata to send to the serial device.
+             * @param {string|Object} data The object containing data and metadata to send to the serial device.
```

Comment:
Per @bberenz's comments, the fix should be in Java, but the API illustrating both will stay here in JS.